powerauth-java-crypto/src/main/java/io/getlime/security/powerauth/crypto/lib/encryptor/ecies/model/EciesPayload.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * PowerAuth Crypto Library
+ * Copyright 2018 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.security.powerauth.crypto.lib.encryptor.ecies.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+/**
+ * The EciesPayload structure represents ECIES payload transmitted over the network.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Value
+@AllArgsConstructor
+public class EciesPayload {
+
+    EciesCryptogram cryptogram;
```

Comment:
~Why is _package private_?~ I see, done by `@Value` https://projectlombok.org/features/Value